### ISSUE TITLE
custom part created in part creator not showing custom textures

### ISSUE THREAD

**Issue by MAriehamn360-Vitrin2** [cuddlyogre/ExportLDraw#50]
after creating a custom lego part with own texture from an image i created in part designer i can only see the main part and the color but not the texture with the image. 
![Screenshot 2024-01-26 191116](https://github.com/cuddlyogre/ExportLDraw/assets/156545899/caee7cf1-c9b9-48f8-bf41-4111a773bc0d)

is there any way to fic this or is it a feature that comes later?

**Comment by cuddlyogre** [comment]
It should be importing the texture. Stud.io's texturing system is fairly complex so it's possible I missed something. 
Would you mind sharing the part so I can troubleshoot?

**Comment by MAriehamn360-Vitrin2** [comment]
cant share io files here. it says its not supported.

**Comment by cuddlyogre** [comment]
I actually just need the exported ldr file that you imported into Blender.

**Comment by MAriehamn360-Vitrin2** [comment]
i cant even send the ldr file. i saw that when i export ldr and opens it in ld view it gets one color without any texture but when pressing on export to studio in part designer it seems to work.

**Comment by MAriehamn360-Vitrin2** [comment]
if i try to export the working custom part from bricklink studio it exports an empty file

**Comment by cuddlyogre** [comment]
Do you see anything if you open the LDR file in notepad?

**Comment by MAriehamn360-Vitrin2** [comment]
wait il take a look

**Comment by MAriehamn360-Vitrin2** [comment]
![Screenshot 2024-01-26 223317](https://github.com/cuddlyogre/ExportLDraw/assets/156545899/e8ef761a-713d-4e1e-be2d-771d8311e154)

Here is the text im getting. 
0 Untitled Model
0 Name:  UntitledModel
0 Author:  
0 CustomBrick
1 15 28.000000 -20.000000 14.000000 1.000000 0.000000 0.000000 0.000000 1.000000 0.000000 0.000000 0.000000 1.000000 m3eef7f7b_2024120_040833.dat

**Comment by cuddlyogre** [comment]
Is there a file named **m3eef7f7b_2024120_040833.dat** in **%LocalAppData%\Stud.io\CustomParts**?

**Comment by MAriehamn360-Vitrin2** [comment]
yes its there. maybe it needs to be also placed in the Ldraw folder in document?

**Comment by cuddlyogre** [comment]
Make sure any parts referenced by your LDR are in the same folder.

This leads me to believe I need to add the custom parts folder to the LDraw paths.

**Comment by MAriehamn360-Vitrin2** [comment]
yes that wold be good to add possibility to add custom and multiple paths.
